### PR TITLE
tests: support for running on single pg version, use in one place

### PR DESCRIPTION
**Pull request by koivunej**
Some tests which are unit test alike do not need to run on different pg versions. Logging test is one of them which I found for unrelated reasons.

### CODE REVIEW

File: test_runner/fixtures/pg_version.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def _missing_(cls, value) -> Optional["PgVersion"]:
         return None
 
 
-DEFAULT_VERSION: PgVersion = PgVersion.V14
+DEFAULT_VERSION: PgVersion = PgVersion.V15
```
Comment (koivunej) [inline test_runner/fixtures/pg_version.py:55]: This is from your original diff, please confirm it has no effect on anything? Or should it be changed in a separate PR if it has an effect?

Comment (bayandin) [reply test_runner/fixtures/pg_version.py:55]: That's ok to keep it here, we don't use `DEFAULT_VERSION` much.

**Review by bayandin — APPROVED** [review]
